tools/lambda-promtail/lambda-promtail/main.go

```diff
@@ -25,19 +26,23 @@ const (
 
 	maxErrMsgLen = 1024
 
-	invalidExtraLabelsError = "invalid value for environment variable EXTRA_LABELS. Expected a comma separated list with an even number of entries. "
+	invalidExtraLabelsError       = "invalid value for environment variable EXTRA_LABELS. Expected a comma separated list with an even number of entries. "
+	invalidEvenExtraHeadersError  = "invalid value for environment variable EXTRA_HTTP_HEADERS. Expected a comma separated list with an even number of entries."
```

Comment:
hey @trc-ikeskin either the name here or on line 89 of the tests needs to change, I think `invalidExtraHeadersError` is enough